README.md

```diff
@@ -80,6 +81,19 @@ All released versions can be found [here](https://github.com/sonatype-nexus-comm
 There are a range of options for installing the R plugin. You'll need to build it first, and
 then install the plugin with the options shown below:
 
+### Easiest Install
+
+Thanks to some upstream work in Nexus Repository (versions newer than 3.15), it's become a LOT easier to install a plugin. To install the `R` plugin, follow these steps:
+
+* Clone this repo and `cd` to the cloned directory location
+* Build the plugin with `mvn clean package -PbuildKar`
+* Copy the `nexus-repository-r-1.0.4-bundle.kar` file from your `target` folder to the `deploy` folder for your Nexus Repository installation.
+
+Once you've done this, go ahead and either restart Nexus Repo, or go ahead and start it if it wasn't running to begin with.
+
+You should see `r (proxy)` in the available Repository Recipes to use, if all has gone according to plan :)
```

Comment:
Trivial (suggestion): Looks like they should also see hosted and group, I had to double check.  Perhaps put "see the R repository types (e.g. `r (proxy)`)" instead?